test/__snapshots__/StatsTestCases.basictest.js.snap

```diff
@@ -4675,3 +4675,25 @@ cacheable modules 2.31 KiB (javascript) 1.37 KiB (webassembly)
     ./node_modules/env.js 34 bytes [built] [code generated]
 webpack x.x.x compiled successfully in X ms"
 `;
+
+exports[`StatsTestCases should print correct stats for worker-public-path 1`] = `
+"With worker public path:
+  asset main-db80575c247d94eb3f64.js 3.53 KiB [emitted] [immutable] (name: main)
```

Comment:
The test is a bit subtle but as you can see, the main.js file has a different hash from the other main.js file